### ISSUE TITLE
Crash on startup - KeyError: 'XDG_SESSION_TYPE'

### ISSUE THREAD

**Issue by mikezila** [Ulauncher/Ulauncher#197]
Ulauncher crashes on startup, with the following output:

```
/usr/lib/python2.7/site-packages/ulauncher/util/desktop/notification.py:1: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
/usr/lib/python2.7/site-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py:6: PyGIWarning: WebKit2 was imported without specifying a version first. Use gi.require_version('WebKit2', '4.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gio, Gtk, WebKit2, GLib
Traceback (most recent call last):
  File "/usr/bin/ulauncher", line 25, in <module>
    main()
  File "/usr/lib/python2.7/site-packages/ulauncher/main.py", line 82, in main
    if is_wayland() and gdk_backend() != 'x11':
  File "/usr/lib/python2.7/site-packages/ulauncher/config.py", line 63, in is_wayland
    return os.environ['XDG_SESSION_TYPE'].lower() == 'wayland'
  File "/usr/lib/python2.7/UserDict.py", line 40, in __getitem__
    raise KeyError(key)
KeyError: 'XDG_SESSION_TYPE'
```
**Steps to reproduce:**
Just try to run `ulauncher`.

**Ulauncher version**: 4.0.3.r0-1 from Arch AUR
**Linux distro**: Antergos/Arch running as a VMware Guest
**Desktop environment**: Openbox + Xorg

**Logs**:
There are no logs, as running `ulauncher -v` crashes the same way.

**Comment by gornostal** [comment]
Hmm looks like there is no XDG_SESSION_TYPE in Openbox + Xorg.
I'll fix that

**Comment by gornostal** [comment]
I made a fix in dev branch.
I don't have Openbox, so could you check if Ulauncher works now by installing dev version? 

```
git clone https://aur.archlinux.org/ulauncher.git && cd ulauncher && makepkg -is
```

**Comment by beanaroo** [comment]
I can confirm the issue is resolved (in i3) in `ulauncher-git`. Thanks for the prompt response!

**Comment by gornostal** [comment]
Cool. Thanks

**Comment by gornostal** [comment]
Fixed in stable release 4.0.4.r1
